src/docs/sphinx/advancedExamples/pygeosxExamples/inSituDataMonitor/AdvancedExample.rst

```diff
@@ -20,34 +20,34 @@ This example requires two input xml files and one python script located at:
 
 .. code-block:: console
 
-  GEOS/examples/pygeosxExamples/hydraulicFractureWithMonitor
+  GEOS/examples/geosPythonInterface/hydraulicFractureWithMonitor
 
 
 ------------------------------------------------------------------
 Description of the case
 ------------------------------------------------------------------
 
 This example is derived from this basic example: :ref:`TutorialHydraulicFractureWithAdvancedXML`, which solves for the propagation of a single hydraulic fracture within a heterogeneous reservoir.
-The pygeosx interface is used to monitor the maximum hydraulic aperture and fracture extents over time.
+The pygeos interface is used to monitor the maximum hydraulic aperture and fracture extents over time.
 
 
 ---------------------------------
 XML Configuration
 ---------------------------------
 
-The input xml file for this example requires some modification in order to work with pygeosx.
+The input xml file for this example requires some modification in order to work with pygeos.
 First, we use the advanced xml input features to include the base problem and override the `table_root` parameter that points to the table files.
 Note that these paths will need to be updated if you run this problem outside of the example directory.
 
-.. literalinclude:: ../../../../../../examples/pygeosxExamples/hydraulicFractureWithMonitor/hydraulicFracture.xml
+.. literalinclude:: ../../../../../../examples/geosPythonInterface/hydraulicFractureWithMonitor/hydraulicFracture.xml
```

Comment:
@rrsettgast - I think that these files are in is: `../../../../../../examples/geosPythonInterfaceExamples`